app/services/support/data_exports/users_export.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+module Support
+  module DataExports
+    class UsersExport < Base
+
+      def id
+        "users"
+      end
+
+      def name
+        "All users"
+      end
+
+      def description
+        "The list of all users with columns: provider_code, provider_name, provider_type, first_name, last_name, email_address"
+      end
+
+      def data
+        result = []
+        User.find_each do |u|
+          if u.providers.empty?
```

Comment:
```ruby
results = RecruitmentCycle.current.providers.includes(:users).map do |provider|
  provider.users.map do |user|
    user_data(user, provider)
  end
end 

results.flatten

```

Otherwise there will be lots of dupes due to recruitment cycle